client/src/components/Modals/ReviewUser/index.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { ReactElement, FC, useState } from 'react';
+import {
+  Button,
+  Grid,
+  Modal,
+  ModalProps,
+  Icon,
+  Form,
+  Input,
+} from 'semantic-ui-react';
+import { IUser, ITeam } from 'ssw-common';
+
+import { updateUser, updateOnboardingStatus } from '../../../api';
+import { useInterests, useTeams } from '../../../contexts';
+import { UserPicture, FieldTag } from '../..';
+import './styles.scss';
+import { getUserFullName, titleCase } from '../../../utils/helpers';
+
+interface ReviewUserProps extends ModalProps {
+  user: IUser;
+}
+
+const ReviewUserModal: FC<ReviewUserProps> = ({ user }): ReactElement => {
+  const [isOpen, setIsOpen] = useState(true);
+  const [formValue, setFormValue] = useState('');
+
+  const { getTeamFromId } = useTeams();
+  const { getInterestById } = useInterests();
+
+  const approveUser = (u: IUser): void => {
+    updateOnboardingStatus(u._id, 'ONBOARDED');
+    setIsOpen(false);
+  };
+
+  const rejectUser = (u: IUser): void => {
+    updateUser(
+      {
+        rejectReasoning: formValue,
+      },
+      u._id,
+    );
+    updateOnboardingStatus(u._id, 'DENIED');
+    setIsOpen(false);
+  };
+
+  return (
+    <Modal
+      size="large"
+      open={isOpen}
+      onOpen={() => setIsOpen(true)}
+      onClose={() => setIsOpen(false)}
+      className="review-user-modal"
+    >
+      <Modal.Header>Review User</Modal.Header>
+      <Modal.Content>
+        <Grid divided="vertically">
+          <Grid.Row columns={2}>
+            <Grid.Column>
+              <div className="user-info">
+                <UserPicture className="review-user-picture" user={user} />
+                <div className="user-info-text">
+                  <b>
+                    {user.preferredName
+                      ? `${user.preferredName} (${user.firstName} ${user.lastName})`
+                      : getUserFullName(user)}
+                  </b>
+                  <br></br>
+                  {user.pronouns.join(', ')}
+                </div>
+              </div>
+            </Grid.Column>
+            <Grid.Column>
+              <Icon name="mail" />
+              {user.email}
+              <br></br>
+              <Icon name="phone" />
+              {user.phone}
+            </Grid.Column>
+          </Grid.Row>
+          <Grid.Row columns={1}>
+            <Grid.Column>
+              <b>Genders:</b> {user.genders.join(', ')}
+              <br /> <b>Races:</b> {titleCase(user.races.join(', '))}
+              {/* Place Holder To Be Updated */}
+              <br /> <b>Neighborhood:</b> Place Holder, update after user
+              onboarding field is added
```

Comment:
User onboarding field? I would change this to say when the neighborhoods field is added to the user model. Also tag this with a TODO like 

```tsx

{/* TODO: ...message... */}

```